File: Listas de exercícios em C/Lista 3 - Sala/Exe 11.c
#include <stdio.h>

int main() {
    int numero;
    int maior = -2147483648;
    int menor = 2147483647;

    for (int i = 0; i < 5; i++) {
        printf("Digite o %dº número inteiro: ", i + 1);
        scanf("%d", &numero);

        if (numero > maior) {
            maior = numero;
        }
        if (numero < menor) {
            menor = numero;
        }
    }

    printf("O maior valor é %d\n", maior);
    printf("O menor valor é %d\n", menor);

    return 0;
}

File: Listas de exercícios em python/Lista 3 - Sala/Exe 11.py
maior = None
menor = None

for i in range(5):
    numero = int(input(f"Digite o {i+1}º número inteiro: "))
    
    if maior is None or menor is None:
        maior = numero
        menor = numero
    else:
        if numero > maior:
            maior = numero
        if numero < menor:
            menor = numero

print(f"O maior valor é {maior}")
print(f"O menor valor é {menor}")